frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -74,28 +74,43 @@ const Map: React.FC<MapProps> = props => {
 
   return (
     <Container fluid={true}>
-      <Row style={{width: "100%", position:"absolute"}}>
+      <Row style={{ width: '100%', position: 'absolute' }}>
         <MapNavBar />
-        <MapFilterBar agencyLookupCodes={props.agencies} propertyClassifications={props.propertyClassifications} onSelectAgency={handleAgencyChanged} onSelectPropertyClassification={handlePropertyClassificationChanged} />
+        <MapFilterBar
+          agencyLookupCodes={props.agencies}
+          propertyClassifications={props.propertyClassifications}
+          onSelectAgency={handleAgencyChanged}
+          onSelectPropertyClassification={handlePropertyClassificationChanged}
+        />
         <LeafletMap
           ref={mapRef}
           center={[props.lat, props.lng]}
           zoom={props.zoom}
           whenReady={() => handleViewportChanged()}
-          onViewportChanged={() => handleViewportChanged()}>
+          onViewportChanged={() => handleViewportChanged()}
+        >
           <TileLayer
             attribution='&amp;copy <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
             url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
           />
 
-          {parcels && parcels.map(parcel => {
-            return (
-              <Marker key={parcel.id} position={[parcel.latitude, parcel.longitude]} onClick={() => handleParcelClick(parcel)} />
-            );
-          })}
+          {parcels &&
+            parcels.map(parcel => {
+              return (
+                <Marker
+                  key={parcel.id}
+                  position={[parcel.latitude, parcel.longitude]}
+                  onClick={() => handleParcelClick(parcel)}
+                />
+              );
+            })}
```

Comment:
To review: another rule that the React team feels improves readability (up for discussion)